resources/js/ziggy.d.ts

```diff
@@ -309,6 +309,27 @@ declare module 'ziggy-js' {
             "required": true
         }
     ],
+    "achievement-set-claim.create": [
+        {
+            "name": "game",
+            "required": true,
+            "binding": "ID"
+        }
+    ],
+    "achievement-set-claim.delete": [
+        {
+            "name": "game",
+            "required": true,
+            "binding": "ID"
+        }
+    ],
+    "achievement-set-claim.update": [
+        {
+            "name": "claim",
+            "required": true,
+            "binding": "ID"
+        }
+    ],
```

Comment:
```
$ sail npm run build

> build
> vite build && vite build --ssr

failed to load config from /var/www/html/vite.config.ts
error during build:
Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'php-array-reader' imported from /var/www/html/vite.config.ts.timestamp-1727622897317-b2c9612ea5222.mjs
    at packageResolve (node:internal/modules/esm/resolve:854:9)
    at moduleResolve (node:internal/modules/esm/resolve:927:18)
    at defaultResolve (node:internal/modules/esm/resolve:1157:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:227:38)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
    at link (node:internal/modules/esm/module_job:86:36)
```

Reply:
If you do `sail npm install`, does that fix `sail npm run build`?

Reply:
it's automatically run by `.hooks/post-merge`. apparently it was failing with a different error. I've nuked my `node_modules` directory and everything seems happier now.